modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelbuilding/v0.1.2.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/emodelbuilding/v0.1.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.1"
+  ],
+  "shapes": [
+    {
+      "@id": "this:EModelBuildingShape",
+      "@type": "sh:NodeShape",
+      "label": "Emodel building shape",
+      "comment": "Get all optimizations for PC CA1",
+      "targetClass": "nsg:EModelBuilding",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.1/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:version",
```

Comment:
The version of the generated .emodel should be attached to emodel entity itself.

Reply:
I think this version is sort of metadata which should not clutter the emodel itself(but we can find it from the activity if needed). So if @wvangeit agrees I can move it to the emodel.

Reply:
We had a chat with @wvangeit and we agreed to have version on the activity.